sqlmesh/engines/commands.py

```diff
@@ -37,6 +37,7 @@ class EvaluateCommandPayload(PydanticModel):
     end: TimeLike
     execution_time: TimeLike
     deployability_index: DeployabilityIndex
+    batch_index: int = 0
```

Comment:
[Nit] I don't think we need a default value here, to make sure that all command's attributes are set explicitly upstream.

Reply:
Fair call, i've removed the default